ramls/check-in.json

```diff
@@ -18,6 +18,20 @@
       "description": "Id of item that has checked in",
       "$ref": "raml-util/schemas/uuid.schema"
     },
+    "itemStatus": {
+      "type": "string",
+      "description": "Item status on check-in"
+    },
+    "requestQueueSize": {
+      "type": "integer",
+      "description": "Request queue size for the item",
+      "minimum": 0
+    },
+    "itemLocationId": {
+      "type": "string",
+      "description": "Effective location of the item",
```

Comment:
I don't think the circulation domain is concerned with how the location for the item is determined, and so the description does not need to include `effective` to refer to the derived nature of the property in inventory.